inference_rules.adoc

```diff
@@ -445,8 +445,10 @@ Non-conforming network submission should be submitted to Open category, under th
 * The QDL is not allowed to pad the data in queries.
 * The QDL is not allowed to cache queries or responses.
 * The QDL is implementing the network function of the LoadGen Node towards the SUT node and handles the required processing. E.G. padding of the payload as required by the network protocol.
+* The QDL should reflect a single SUT to the LoadGen. LoadGen operates with a single SUT. 
 * The Name method's return value must contain the substring "Network SUT".
 * The Name method's implementation must include at least one round trip over the network. The Name method must not return until the round trip is complete.
+* The QDL should serially get the names from all the Nodes of the SUT and aggregate the responses in the Name Method after getting response from all Nodes. Each Node must have a unique Name.   
```

Comment:
```suggestion
* The QDL must query each SUT Node for its name and aggregate the responses in the Name Method. Each SUT Node must have a unique name.   
```

Reply:
One clarification please - the word serially is not appearing in your text. Do we care if in multiple nodes we query in parallel? Do we consider the latency accumulation as an additional information or just want to see it is Network > local?

Reply:
OK I accept your suggestion